mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Eredis.Mixfile do
   def project do
     [
       app: :eredis,
-      version: "1.3.0-nordix",
+      version: "1.3.1-nordix",
```

Comment:
now that this will be the official Eredis distribution, we can drop `-nordix` from the version

Reply:
Yes, we can tag again, but first we can add some link to Hex in README and maybe similar changes. PR is welcome. :-)